src/main/kotlin/app/revanced/patches/all/location/hide/HideMockLocationPatch.kt

```diff
@@ -0,0 +1,57 @@
+@file:Suppress("unused")
+
+package app.revanced.patches.all.location.hide
+
+import app.revanced.patcher.patch.annotation.Patch
+import app.revanced.patcher.util.proxy.mutableTypes.MutableMethod
+import app.revanced.patches.all.misc.transformation.BaseTransformInstructionsPatch
+import app.revanced.patches.all.misc.transformation.IMethodCall
+import app.revanced.patches.all.misc.transformation.Instruction35cInfo
+import app.revanced.patches.all.misc.transformation.filterMapInstruction35c
+import com.android.tools.smali.dexlib2.iface.ClassDef
+import com.android.tools.smali.dexlib2.iface.Method
+import com.android.tools.smali.dexlib2.iface.instruction.Instruction
+
+@Patch(
+    name = "Hide Mock Location",
+    description = "Prevent app from knowing whether the location is mocked",
```

Comment:
```suggestion
    description = "Spoof the app when mocking the location.",
```

@LisoUseInAIKyrios maybe you have a better description.